lib/vimeo.js

```diff
@@ -582,3 +650,38 @@ Vimeo.prototype._performTusUpload = function (
   upload.url = attempt.upload.upload_link
   upload.start()
 }
+
+/**
+ * Take an upload attempt and perform the actual upload via tus.
+ *
+ * https://tus.io/
+ *
+ * @param {File}      file              File you wish to upload.
+ * @param {integer}   fileSize          Size of the file that will be uploaded.
+ * @param {Object}    attempt           Upload attempt data.
+ * @param {Function}  completeCallback  Callback to be executed when the upload completes.
+ * @param {Function}  progressCallback  Callback to be executed when the upload progress is updated.
+ * @param {Function}  errorCallback     Callback to be executed when the upload returns an error.
+ */
+Vimeo.prototype._performTusFileUpload = function(
+  file,
+  fileSize,
+  attempt,
+  completeCallback,
+  progressCallback,
+  errorCallback
+) {
+  var upload = new tus.Upload(file, {
```

Comment:
Similar to the other comment, we can eliminate this method and merge it with `_performTusUpload` by detecting if `file` is a blob or string, and running it through `fs.createReadStream` depending.

Reply:
Great! It's make sence, I will make the change